README.md

```diff
@@ -0,0 +1,40 @@
+
+| Title | KOBSearch |
+|--- | ---|
+| Version | TBD |
+
+
+### Abstract   
+
+The KOBSearch is a search service that provides a registered user with the information about the organizations (or members of the organization) that are operating in the city of Cochin and registered with KochiOrgBook (KOB). KOB is like a directory that will contain the verifiable data about the organizations thus enabling trusted digital verification service.
+
+### Dependent Projects
+
+KOBSearch is a UI plug-in that will run on [KOBConnect](https://github.com/hyperledgerkochi/KOBConnect) Wallet App.
+
+### Motivation
+
+In the existing scenario, there are many possibilites for crimes (like document forgery) and other fraudulent activities. This makes it difficult for a citizen to confirm the original identity of various associations or people and thereby fall into the traps set by cheaters. That means, there is an element of trust that is missing.  Inorder to avoid this, finding authentic and authoritative data about people or orgaiztaion, has become a crucial factor. KOBSearch helps to achieve this.
```

Comment:
- "Crimes", is more generic. Try using something more specific.
- "orgaiztaion" - Clear the typo
- "KOBSearch helps to achieve this" - This line is incorrect. The search just finds the data. authentic data is made with the help of other project.